appveyor.yml

```diff
@@ -7,83 +7,11 @@ platform: x86
 clone_folder: c:\projects\azure-webjobs-sdk
 environment:
   IncludeBuildNumberInVersion: 1
-install:
-- ps: >-
-    $isPr = Test-Path env:APPVEYOR_PULL_REQUEST_NUMBER
-
-
-    if (-not $isPr) {
-        $ctx = New-AzureStorageContext $env:FILES_ACCOUNT_NAME $env:FILES_ACCOUNT_KEY
-        $queue = Get-AzureStorageQueue "jenkins" -Context $ctx
-
-        $messageBody = "$env:APPVEYOR_BUILD_VERSION;$env:APPVEYOR_REPO_COMMIT;$env:IncludeBuildNumberInVersion"
-        $message = New-Object -TypeName Microsoft.WindowsAzure.Storage.Queue.CloudQueueMessage -ArgumentList $messageBody
-        $queue.CloudQueue.AddMessage($message)
-
-        "jenkins signed build:   https://zenkins.redmond.corp.microsoft.com/view/Functions/job/Functions_webjobs_sdk/' build #appveyor $env:APPVEYOR_BUILD_VERSION"
-        "artifacts azure files:  https://$env:FILES_ACCOUNT_NAME.file.core.windows.net/jenkins/builds/$env:APPVEYOR_BUILD_VERSION"
-        "signed build onenote:   https://microsoft.sharepoint.com/teams/AzureWebjobs/_layouts/OneNote.aspx?id=%2Fteams%2FAzureWebjobs%2FSiteAssets%2FAzureWebjobs%20Notebook&wd=target%28Signed%20Sdk%20Build.one%7C0C569AE2-E052-428D-B3C1-8421ED9B1CF8%2F%29"
-    }
 build_script:
-- cmd: msbuild "WebJobs.proj" /t:BuildTestBinaries /logger:"C:\Program Files\AppVeyor\BuildAgent\Appveyor.MSBuildLogger.dll" /property:Configuration=Release;Platform=AnyCPU;OutputPath=%APPVEYOR_BUILD_FOLDER%\bin
-test_script:
-- cmd: "vstest.console /logger:Appveyor /TestAdapterPath:bin bin/Microsoft.Azure.WebJobs.Host.UnitTests.dll bin/Microsoft.Azure.WebJobs.Host.FunctionalTests.dll bin/Microsoft.Azure.WebJobs.ServiceBus.UnitTests.dll bin/Dashboard.UnitTests.dll \n\nvstest.console /logger:Appveyor /TestAdapterPath:bin bin/Microsoft.Azure.WebJobs.Host.EndToEndTests.dll"
-artifacts:
-- path: bin\Binaries\*.*
-  name: Binaries
+- cmd: msbuild "WebJobs.proj" /t:GetBitsToSign;BuildTestBinaries /logger:"C:\Program Files\AppVeyor\BuildAgent\Appveyor.MSBuildLogger.dll" /property:Configuration=Release;Platform=AnyCPU
```

Comment:
It makes sense for testing here, but it feels like we should disable it for regular PRs. I may be paranoid, but allowing someone to submit a PR that contains assemblies that will get passed to our signing server made me nervous.